dbms/include/DB/DataStreams/JSONCompactRowOutputStream.h

```diff
@@ -14,7 +14,7 @@ namespace DB
 class JSONCompactRowOutputStream : public JSONRowOutputStream
 {
 public:
-	JSONCompactRowOutputStream(WriteBuffer & ostr_, const Block & sample_, bool write_statistics_);
+	JSONCompactRowOutputStream(WriteBuffer & ostr_, const Block & sample_, bool write_statistics_, bool force_quoting_);
```

Comment:
On the other hand, parameter with such name is too specific for interface function.
However, I don't have other suggestions.